src/global/Styles.js

```diff
@@ -760,6 +760,24 @@ const styles = {
   disabledButtonTitle: {
     color: Colors.BitnationLightGrayColor,
   },
+
+  dAppMessage: {
+    width: 250,
```

Comment:
I think we shouldn't use constant width, since device width may be different and/or, message can be small and therefore don't need too be that big. Furthermore, since we pass custom view inside default Bubble component on the chat, it will likely automatically handle its size (need to be tested).

Reply:
The dAppMessage custom view has a larger width than the default bubble component, as per the designs. I'm not sure if it will automatically handle the width.

Reply:
I mean, fixed width is not good in any case, because there may be devices with width smaller than 250 pixels. I would go without setting any specific sizes.

Reply:
Yeah sure, will do that

Reply:
Ok, removing the specific size. Thanks